packages/next-swc/crates/next-core/src/next_font/google/mod.rs

```diff
@@ -169,11 +168,14 @@ impl ImportMappingReplacement for NextFontGoogleCssModuleReplacer {
         let options = font_options_from_query_map(*query_vc, font_data);
         let stylesheet_url = get_stylesheet_url_from_options(options, font_data);
         let request_hash = get_request_hash(*query_vc);
-        let scoped_font_family =
-            get_scoped_font_family(FontFamilyType::WebFont.cell(), options, request_hash);
+        let scoped_font_family = get_scoped_font_family(
+            FontFamilyType::WebFont.cell(),
+            options.await?.font_family.to_owned(),
```

Comment:
Can we pass the options into the function? That way the outer `result` doesn't depend on the value of options.